src/api/config/application.rb

```diff
@@ -2,8 +2,14 @@
 
 require 'rails/all'
 
-# Assets should be precompiled for production (so we don't need the gems loaded then)
-Bundler.require(*Rails.groups(assets: ['development', 'test']))
+gemfile_in = File.expand_path('../Gemfile.in', __dir__)
```

Comment:
Sorry but this distro packaging specific thing should be in a distro specifc patch in a distro specific package. We do can't support this in any way.

Reply:
So then, would you be okay with me deleting SUSE-isms that aren't useful for everyone else from OBS? For example, the creation of rcFOO links that don't work anywhere else and are completely unnecessary now that we're in systemd era?

Reply:
But yeah, I can remove that patch from this list...

Reply:
Dropped.

Reply:
> So then, would you be okay with me deleting SUSE-isms that aren't useful for everyone else from OBS? For example, the creation of rcFOO links that don't work anywhere else and are completely unnecessary now that we're in systemd era?

Sure, I would be fine with this.

Reply:
Done.